addons/godot-xr-tools/player/player_body.gd

```diff
@@ -421,7 +421,7 @@ func move_player(p_velocity: Vector3) -> Vector3:
 			var with = get_slide_collision(idx)
 			var obj = with.get_collider()
 
-			if obj.is_class("RigidBody3D"):
+			if obj!=null and obj.is_class("RigidBody3D"):
```

Comment:
Just change this to `if obj and obj.is_class("RigidBody3D")`, reads easier and is correct gdscript syntax.